backend/icpy/agent/tools/atlascloud/itv_tool.py

```
@@ -0,0 +1,610 @@
+            video_url = result.video_url
+            if not video_url:
+                return ToolResult(
+                    success=False,
+                    data={"status": result.status},
+                    error=f"Video generation completed but no output URL found. Status: {result.status}"
+                )
+            
+            logger.info(f"[AtlasCloudITV] Video generated: {video_url}")
+            
+            # Download video
+            logger.info(f"[AtlasCloudITV] Downloading video...")
```

Ruff (0.14.14)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)